app/controllers/api/v1/items_controller.rb

```diff
@@ -46,7 +46,7 @@ def destroy
 
         private
         def item_params
-          params.permit(:menu_id, :item_name, :price, :category)
+          params.permit(:menu_id, :item_name, :price, :category, :description, :cals)
```

Comment:
I like the addition of the calories column. When it comes to food the more info we can provide to the users the better.